express-server/middleware/userIdMiddleware.js

```diff
@@ -0,0 +1,24 @@
+const { v4 } = require('uuid');
+
+function parseCookies(request) {
+  var list = {},
```

Comment:
no copy paste. plz write this by yourself. (hint. you don't need this code..) you can access the cookies directly